src/settings/SettingsDetails.js

```diff
@@ -21,7 +21,7 @@ export default class SettingsDetails extends Component {
 					<LabeledText
 						id="name"
 						label="Name"
-						value={settings.name}
+						value={settings.name || ''}
```

Comment:
Should we also send null to the server then, when user updates and the value is ''?